README.md

```diff
@@ -6,17 +6,30 @@ View the [demo](https://adobedocs.github.io/dev-site-documentation-template/) ru
 
 ## Where to ask for help
 
-The slack channel #adobeio-onsite-onboarding is our main point of contact for help. Feel free to join the channel and ask any questions. 
+The slack channel #adobeio-onsite-onboarding is our main point of contact for help. Feel free to join the channel and ask any questions.
 
-## How to develop
+## How to build locally
+
+This site uses the Yarn package manager to manage dependencies and run scripts. Open a terminal at the project root and enter these commands in the following order:
+
+1. Install Yarn globally: `npm i -g yarn`.
+2. Install site dependencies: `yarn`.
+3. Build site locally: `yarn dev`.
+
+## How to fix MDX build errors
+
+To fix common markdown errors within MDX files, open a terminal at the project root and enter the following command:
 
-For local development, simply use :
 ```
-$ npm install
-$ npm run dev
+yarn fix-markdown src/pages
```

Comment:
Does this include markdown linting errors or does it refer to the use of MDX tags within markdown?

Reply:
Just fixes the problematic markdown syntax. Nothing to do with linting, sorry to say.